grails-app/conf/application.yml

```diff
@@ -101,6 +101,7 @@ serverName: https://sds.ala.org.au
 security:
   cas:
     appServerName: https://sds.ala.org.au
+
```

Comment:
Any reason why this is here?

Reply:
My bad , removed in 5b4ffac97b318e17cc792f018654c097bde50fb8